css/style.css

```diff
@@ -37,23 +37,188 @@ Developer @GA in NYC
   float: left;
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
+.categories {
+  display: block;
+  font-weight:300;
+  font-size:24px;
+}
+.nav li {
+  display: inline-block;
+  color: #f9e42e;
+  font-weight: 600;
+  font-size: 32px;
+  padding: 8px;
+}
+.nav ul {
+  list-style: none;
+  overflow: auto;
+  width: 600px;
+  margin: 0 auto;
+
+}
+.signup {
+  float: left;
```

Comment:
Or alternatively, you can do what I said above but instead of adding the `clear: both` rule to `.signup {}`, you can simply add your pre-existing `clear` class to the `<div>` that contains the class `signup`. In other words:

`<div class="signup clear">...</div>`